CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Issues
+
+Issues have three types: bug, feature request, and support. When reporting a bug, you must include
+the operating system used, any relevant information about the tech stack, and the feature flags
+used. 
+
+# Pull Requests
+
+Pull requests should be named with the format `crate: short description of change`, and should use
+lower case letters. Always make a pull request from a new branch, that is named similarly, but with
+only a few words: `crate-short-description`. If adding a feature or enhancement, use the term `add`
+or something sufficiently similar. If fixing a bug, use the term `fix`, or something sufficiently
+similar. Avoid force-pushing to a pull request branch, as this erases review comment history.
+
+# Labeling
+
+If you can, you must label your issues and pull requests appropriately. This includes adding a label
```

Comment:
This is contradictory saying "can" and then following it with "must" imo. It also doesn't define what "can" means.

```suggestion
If an applicable labels exist, then you must label your issues and pull requests appropriately. This includes adding a label
```

Reply:
I meant that if you are part of the org. I can reword it to `If you are able`

Reply:
That sounds good to me.